### PR TITLE
Adding new virtualiation roles

### PR DESCRIPTION
**Pull request by e0d**
Stub roles for container virtualization and Docker.  Docker specific common tasks, say, setting up DNS search path to work with Docker networking will occur here.

@cpennington @feanil

### CODE REVIEW

File: playbooks/roles/docker/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# Dependencies:
+#
+# 
+# Example play:
+#
+#
+
+- name: stub ansible task
+  debug: msg="This is a stub task created by the ansible-role role"
+  notify: notify me
```
Comment (cpennington) [inline playbooks/roles/docker/tasks/main.yml]: Leave this out?

**Comment by cpennington** [comment]
I'm a bit confused. Are you adding this just to make it obvious where docker-common tasks should go in the future? Or are you going to be adding some tasks as part of this PR?

**Comment by e0d** [comment]
There will be separate PRs to move tasks here, hence stub.  This existed in a prior branch, but was lost in a rebase.  The analog is `vhost --> aws`.

**Comment by cpennington** [comment]
I see.

**Comment by e0d** [comment]
I've removed the boiler plate, would like to merge so I can start fixing the commented out Dockerfiles.

**Comment by feanil** [comment]
:+1:

**Pull request merged**